project/PekkoDisciplinePlugin.scala

```diff
@@ -164,7 +164,7 @@ object PekkoDisciplinePlugin extends AutoPlugin {
     "-Ywarn-numeric-widen",
     "-Yno-adapted-args",
     "-deprecation",
-    "-Xlint",
+    "-Xlint:-infer-any",
```

Comment:
This enables no lints. `-Xlint -Xlint:-infer-any` or `-Xlint:-infer-any,_` with underscore to mean all. (Not sure if that was intended here.)

Reply:
Thanks, fixing now